janusgraph-solr/src/main/java/org/janusgraph/diskstorage/solr/SolrIndex.java

```diff
@@ -515,6 +516,13 @@ private QueryResponse runCommonQuery(RawQuery query, KeyInformation.IndexRetriev
                                 .setIncludeScore(true)
                                 .setStart(query.getOffset())
                                 .setRows(query.hasLimit() ? query.getLimit() : maxResults);
+        for(final Parameter parameter: query.getParameters()) {
+            if (parameter.value() instanceof String[]) {
+                solrQuery = solrQuery.setParam(parameter.key(), (String[]) parameter.value());
```

Comment:
I guess I was doing this out of safety since while unlikely, if setParam changes its implementation and returns a new object then it would be broken.  I'll update it.